app/views/courses/_description_content.html.erb

```diff
@@ -0,0 +1,127 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h3 class="govuk-heading-m govuk-!-font-size-27">
+      <%= link_to 'About this course', manage_ui_course_page_url(provider_code: @provider[:provider_code], course_code: @course.attributes[:course_code]) + '/about', class: 'govuk-link' %>
+    </h3>
+    <dl class="govuk-summary-list govuk-summary-list--short">
+      <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key">
+          About this course
+        </dt>
+        <dd class="govuk-summary-list__value govuk-summary-list__value--truncate <%= 'course-parts__fields__value--empty' if @course.attributes[:about_course].nil? %>" data-qa="course__about">
+          <%= @course.attributes[:about_course] || 'Empty' %>
+        </dd>
+      </div>
+
+      <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key">
+          Interview process (optional)
+        </dt>
+        <dd class="govuk-summary-list__value govuk-summary-list__value--truncate <%= 'course-parts__fields__value--empty' if @course.attributes[:interview_process].nil? %>" data-qa="course__interview_process">
+          <%= @course.attributes[:interview_process] || 'Empty' %>
+        </dd>
+      </div>
+
+      <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key">
+          How school placements work
+        </dt>
+        <dd class="govuk-summary-list__value govuk-summary-list__value--truncate <%= 'course-parts__fields__value--empty' if @course.attributes[:how_school_placements_work].nil? %>" data-qa="course__placements_info">
+          <%= @course.attributes[:how_school_placements_work] || 'Empty' %>
+        </dd>
+      </div>
+    </dl>
+
+    <h3 class="govuk-heading-m">
+      <% if @course.attributes[:funding] == 'fee' %>
+        <%= link_to 'Course length and fees', manage_ui_course_page_url(provider_code: @provider[:provider_code], course_code: @course.attributes[:course_code]) + '/fees-and-length', class: 'govuk-link' %>
+      <% else %>
+        <%= link_to 'Course length and salary', manage_ui_course_page_url(provider_code: @provider[:provider_code], course_code: @course.attributes[:course_code]) + '/salary', class: 'govuk-link' %>
```

Comment:
Just to double check, is this also what shows for courses that are Apprenticeships? (Example course if you need one: https://find-postgraduate-teacher-training.education.gov.uk/course/2BZ/39NZ)

Reply:
That's correct. In the c# ui we have a method that checks for salary or apprenticeships so here I just check for the alternative i.e. `fee`